src/components/OcSidebarNav.vue

```diff
@@ -71,7 +71,7 @@ export default {
   padding: var(--oc-space-large) 0 var(--oc-space-medium);
   width: var(--oc-size-width-medium);
 
-  &-header {
+  &-figure {
```

Comment:
Since you didn't change the class in line 10 in `src/components/OcSidebarNav.spec.js` this change is not needed